.changeset/neat-lights-worry.md

```diff
@@ -0,0 +1,13 @@
+---
+"@fluidframework/runtime-definitions": minor
+"@fluidframework/test-runtime-utils": minor
+---
+---
+"section": deprecation
+---
+
+The function `process` has been deprecated on the following interface - `IFluidDataStoreChannel`. It has also been deprecated on the following mock test class - `MockFluidDataStoreRuntime`.
```

Comment:
Similar to above. I think this one does want the simplification into a single sentence to be a better heading.
```suggestion
The `process` function on `IFluidDataStoreChannel` and `MockFluidDataStoreRuntime` is now deprecated
```